pages/pinner-award.md

```diff
@@ -0,0 +1,51 @@
+title: The John Pinner Award
+---
+<h2>What?</h2>
```

Comment:
Not really important, but given that this is Markdown, you could use `## What?` etc.  (Don't worry about making the change this time!)